### ISSUE TITLE
Typescript Client for UserOfficeWebService rest service

### ISSUE THREAD

**Issue by TCMeldrum** [UserOfficeProject/user-office-project-issue-tracker#945]
### What is the problem and why is it a problem

Like we have the user office web service client for soap: https://github.com/UserOfficeProject/user-office-lib/tree/master/packages/uows-client-generator we might want to create one for the rest interfaces as well as this is used in a few different localations. It looks like it should be easy as Users are using Open API and there are library to create the client for us for example: https://www.npmjs.com/package/openapi-typescript-codegen.

**Comment by TCMeldrum** [comment]
Blocked by https://github.com/isisbusapps/Users/issues/3459

**Comment by mutambaraf** [comment]
Should this also include making changes to our mock server .

**Comment by joshhdawes** [comment]
Duplicate of other issues
